infrastructure/kzg/src/test/java/tech/pegasys/teku/kzg/KZGTest.java

```diff
@@ -118,12 +129,6 @@ public void testVerifyPointEvaluationPrecompile() {
     final KZGProof kzgProof = new KZGProof(emptyCommitment);
     assertThat(kzg.verifyKzgProof(kzgCommitment, Bytes32.ZERO, Bytes32.ZERO, kzgProof)).isTrue();
     assertThat(kzg.computeAggregateKzgProof(Collections.emptyList())).isEqualTo(kzgProof);
-    assertThat(
```

Comment:
I don't understand why all tests were removed
Also it would be great if we could test new Minimal Preset somewhere